pocs/camera/sbig.py

```diff
@@ -60,12 +58,19 @@ def CCD_set_point(self):
     @CCD_set_point.setter
     def CCD_set_point(self, set_point):
```

Comment:
Seeing this repeated makes me wonder if we should have a `CCDCamera` that inherits from the base and then the SBIG and FLI inherit from `CCDCamera`. Not sure if that would help though.

Reply:
I don't think they've got a lot of common code, now that I've moved a bunch of stuff into the base class.  Worth considering, though.